src/app/public/modules/autonumeric/autonumeric.directive.ts

```diff
@@ -52,7 +52,12 @@ const SKY_AUTONUMERIC_VALIDATOR = {
 };
 // tslint:enable
 
-@Directive({
+  /**
+   * Wraps the autoNumeric jQuery plugin to format any type of number, including currency.
```

Comment:
jQuery shouldn't be mentioned. Also, it's not really a "plugin" anymore; more akin to a JavaScript "utility" or service.